### PR TITLE
🎒 Unify network config

### PR DESCRIPTION
**Pull request by jodator**
Related: #1845, #1812

Waiting for: #1849

### CODE REVIEW

**Comment by jodator** [comment]
@krupka4soft Please take a look :)

File: packages/ui/src/app/config/network.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  process.env.REACT_APP_OLYMPIA_TESTNET_QUERY_NODE || 'https://olympia-dev.joystream.app/query/server/graphql'
+const OLYMPIA_TESTNET_QUERY_NODE_SOCKET =
+  process.env.REACT_APP_OLYMPIA_TESTNET_QUERY_NODE_SOCKET || 'wss://olympia-dev.joystream.app/query/server/graphql'
+const MEMBERSHIP_FAUCET_URL = process.env.REACT_APP_MEMBERSHIP_FAUCET_URL || 'http://localhost:4000/register'
+
+export const QUERY_NODE_ENDPOINT_SUBSCRIPTION: Record<NetworkType, string> = {
+  local: 'ws://localhost:8081/graphql',
+  'local-mocks': 'ws://localhost:8081/graphql',
+  'olympia-testnet': OLYMPIA_TESTNET_QUERY_NODE_SOCKET,
+}
```
Comment (DevNikita16) [inline packages/ui/src/app/config/network.ts:15]: Nice 👍

**Review by DevNikita16 — APPROVED** [review]

File: packages/ui/src/app/config/network.ts (hunk 1 above)
Comment (DevNikita16) [reply packages/ui/src/app/config/network.ts:15]: Also liked that you separated configs by domains, could be useful in future

**Comment by jodator** [comment]
@krupka4soft Please take a look when you'll have time. Merging it now, but feel free to create a follow-up issue if you find something that you'd like to change.

**Pull request merged**